Source/SwiftLintFramework/Rules/RuleConfigurations/BalancedXCTestLifecycleConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+public struct BalancedXCTestLifecycleConfiguration: RuleConfiguration, Equatable {
+    public private(set) var severityConfiguration = SeverityConfiguration(.warning)
+    public private(set) var testParentClasses: Set<String> = ["XCTestCase"]
+    
+    public var consoleDescription: String {
+        return severityConfiguration.consoleDescription +
+            ", test_parent_classes: [\(testParentClasses)]"
```

Comment:
The brackets are superfluous. Sorting the set makes the output deterministic:
```suggestion
            ", test_parent_classes: \(testParentClasses.sorted())"
```

Reply:
So I added `QuickSpec` to all the defaults, and rolled the new ones up, with typealiases for the names you'd expect.

But I bodged rebasing, so I just moved to a new branch and PR -  #4262